scripts/test/SceneBuilder.ps1

```diff
@@ -0,0 +1,300 @@
+   $UnityPath = ${Env:${{ parameters.UnityVersion }}}
```

Comment:
Nit: Add a comments/header type thing here explaining what the purpose of this script is

See other scripts like:

https://github.com/microsoft/MixedRealityToolkit-Unity/blob/mrtk_development/scripts/ci/validatecode.ps1